src/server/terrain/simple_structures/SBBGen.zig

```diff
@@ -0,0 +1,120 @@
+const std = @import("std");
+
+const main = @import("main");
+const terrain = main.server.terrain;
+const GenerationMode = terrain.biomes.SimpleStructureModel.GenerationMode;
+const CaveMapView = terrain.CaveMap.CaveMapView;
+const CaveBiomeMapView = terrain.CaveBiomeMap.CaveBiomeMapView;
+const sbb = terrain.structure_building_blocks;
+const Blueprint = main.blueprint.Blueprint;
+const ZonElement = main.ZonElement;
+const Neighbor = main.chunk.Neighbor;
+const ServerChunk = main.chunk.ServerChunk;
+const NeverFailingAllocator = main.heap.NeverFailingAllocator;
+const parseBlock = main.blocks.parseBlock;
+const hashInt = main.utils.hashInt;
+const hashCombine = main.utils.hashCombine;
+
+pub const id = "cubyz:sbb";
+pub const generationMode = .floor;
+
+const SubstitutionMap = std.AutoHashMapUnmanaged(u16, u16);
```

Comment:
This is duplicated, as importing it from blueprint causes stack overflow in `refAllDeclsRecursiveExceptCImports` due to reference cycle.